rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -206,6 +206,10 @@ List<UserJobData> jobsInfoListPost(@HeaderParam("sessionid") String sessionId, L
      *            PENDING_TASKS,RUNNING_TASKS,FINISHED_TASKS,FAILED_TASKS,FAULTY_TASKS,IN_ERROR_TASKS
      *            Each parameter must end with _d for descending order or _a for ascending.
      *            Default value is: "STATE_a,ID_d"
+     * @param submittedTimeLessThan
+     *             Include only jobs with submittedTime less than submittedTimeLessThan
+     * @param submittedTimeGreater
+     *             Include only jobs with submittedTime greater than submittedTimeGreater
```

Comment:
Include only jobs with a submittedTime after the given EPOCh time.

You also need to reverse these two parameters, it makes more sense to have the beginning of the window before the end of the window.